build/static_dist.js

```diff
@@ -112,12 +112,21 @@ function addPackages(siteConfig, buildFolder) {
 
 		return readFile(path.join(fullBuildFolderPath, "package.json")).then(function(packageContents){
 			var json = JSON.parse(packageContents);
+			var filePathedDeps = siteConfig.html.dependencies;
 
-			json.dependencies = _.assign(json.dependencies || {},siteConfig.html.dependencies);
+			for(var depName in siteConfig.html.dependencies) {
+				var possibleFilePath = siteConfig.dependencies[depName];
+
+				if (_.startsWith(possibleFilePath,'file:')) {
```

Comment:
Sorry, I should’ve been more clear; any reason not to use the native String APIs here? Not a big deal, just asking if there’s a reason to use Lodash.